contracts/deployments/strategies/11155111.json

```diff
@@ -0,0 +1,7 @@
+{
+  "strategies": [
+    "0x424246eF71b01ee33aA33aC590fd9a0855F5eFbc",
```

Comment:
Again, it would be good to document which ones these are. I guess sETH.

Not sure where